lib/tasks/forms.rake

```diff
@@ -43,6 +43,13 @@ def move_forms(form_ids, group_id)
     end
 
     group_form.update!(group:)
+
+    next unless form.organisation && form.organisation != group.organisation
+
+    Rails.logger.info "forms:move: updating #{fmt_form(form)} organisation from #{form.organisation.name} to #{group.organisation.name}"
+
+    form.organisation = group.organisation.name
```

Comment:
This doesn't seem to work on my machine - the task logs that it's updating the organisation, but the organisation doesn't actually change. 

I think this line might be the issue - you might need to send the ID rather than the name. I can see in the API log that the `PUT` request being sent contains the old organisation ID
```suggestion
    form.organisation_id = group.organisation.id
```

Reply:
Ah. Thanks for catching that! I wonder why my test didn't work 🤔

Reply:
That's weird!

Reply:
I've updated the tests to use our custom `have_been_updated` matcher.

Reply:
All working for me locally now 👍